pkg/dockerbuild/spec.go

```diff
@@ -174,6 +176,14 @@ func (h HostPath) JoinHost(p HostPath) HostPath {
 	return h.Join(string(p))
 }
 func (h HostPath) ToImage() ImagePath {
+	if runtime.GOOS == "windows" {
+		// convert windows path with volume to a unix path, i.e c:\some\path -> /c/some/path
+		volume := filepath.VolumeName(string(h))
+		if len(volume) == 2 && volume[1] == ':' {
+			return ImagePath("/" + string(volume[0]) + filepath.ToSlash(strings.TrimPrefix(string(h), volume)))
```

Comment:
Since we know `volume` is 2 bytes, how about:

```suggestion
			return ImagePath("/" + string(volume[0]) + filepath.ToSlash(h[2:]))
```

Reply:
clever!